### PR TITLE
Added example for VAD based mic streaming in NIM language

### PR DESCRIPTION
**Pull request by eagledot**

### CODE REVIEW

**Comment by lissyx** [comment]
Care to share what is nim ?

**Comment by eagledot** [comment]
NIM is a recent language, although already reached stable version >1.0 and compile to C language.
 Did you mean to ask  to share more about NIM language?

**Comment by lissyx** [comment]
Well you open a PR about a new language, it'd be nice for people to have a few resources to know more about it.

**Comment by eagledot** [comment]
I could not think of where to share more  information about language itself since this repo is focused on examples. I found some  references to external repositories for language like  V,so i opened the PR.
I would be be happy to add more information if you have some suggestions. Or should i move this to an external repository completely ?

**Comment by lissyx** [comment]
You could just share more in the PR comments itself

**Comment by eagledot** [comment]
More about Nim language and included tools is described at:
[Nim-lang](https://en.wikipedia.org/wiki/Nim_%28programming_language%29)

Resources for learning nim language are summarized quite nicely at:
[https://nim-lang.org/learn](https://nim-lang.org/learn.html) .

Documentation:  [https://nim-lang.org/documentation](https://nim-lang.org/documentation.html)

There is very active  Gitter [channel](https://gitter.im/nim-lang/Nim)

**Comment by eagledot** [comment]
I accidentally closed it :)

File: nim_mic_vad_streaming/linux_nim_vad_streaming/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+wget https://github.com/mozilla/DeepSpeech/releases/download/v0.7.0/deepspeech-0.7.0-models.pbmm
+wget https://github.com/mozilla/DeepSpeech/releases/download/v0.7.0/deepspeech-0.7.0-models.scorer
+```
+
+
+## Installation
+
+1. Install Nim bindings for deespeech version-0.7.0 .
+```nim
+nimble install https://gitlab.com/eagledot/nim-deepspeech@0.7.0
```
Comment (lissyx) [inline nim_mic_vad_streaming/linux_nim_vad_streaming/README.md:24]: If you have bindings, please make a PR on https://github.com/mozilla/DeepSpeech to document them

**Comment by lissyx** [comment]
Thanks for that @eagledot can you provide something to make that testable?

**Comment by eagledot** [comment]
Thanks @lissyx   for a fast review.
I will make a PR on [https://github.com/mozilla/DeepSpeech](https://github.com/mozilla/DeepSpeech) for bindings.
Should i add some tests in for example you reviewed?

**Comment by eagledot** [comment]
I get your point completely.Should have written a better description in the first place.
This was my first major PR ,guess didn't think through much.
I will add some tests and make a PR.

**Pull request merged**